.github/workflows/update_projects.yml

```
@@ -0,0 +1,42 @@
+    - cron: '30 18 * * 5' # 18:30 UTC Friday = 00:00 IST Saturday
+  workflow_dispatch: # Allow manual triggering
+
+jobs:
+  update-projects:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # Need write permission to commit
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)